utils/token.ts

```diff
@@ -0,0 +1,2 @@
+export const shortenAddress = (address: string) =>
+  `${address.slice(0, 4)}...${address.slice(-3)}`;
```

Comment:
Let's make it 5.
Or 6 and -4.

```suggestion
  `${address.slice(0, 5)}...${address.slice(-3)}`;
```

Reply:
Addresses don't give much info, it's just a small set of numbers to be recognisable as an address and maybe in some cases you identify it, like some of your wallets you know they start in some digits.

Either way I increase it a bit on both sides.